docs/app/components/grid/grid-demo.component.html

```diff
@@ -12,16 +12,17 @@ <h5>About</h5>
 
             <hc-tile>
                 <h5>Example </h5>
-                <table [hc-grid]="rows" [gridOptions]="gridOptions" #tableExample>
+                <table [hc-grid]="rows" [gridOptions]="gridOptions" class="hc-table-borders" #tableExample>
                     <thead>
                         <tr>
+                            <th hc-master-checkbox></th>
                             <th hc-sortable="heading1">Heading 1</th>
                             <th hc-sortable="heading2">Heading 2</th>
                             <th hc-sortable="heading3">Heading 3</th>
                         </tr>
                     </thead>
                     <tbody>
-                        <tr *ngFor="let row of tableExample.rows">
+                        <tr *ngFor="let row of tableExample.rows" [hc-selectable]="row" [showCheckbox]="true">
```

Comment:
I like that this interface is flexible, but it also feels a bit complex. I would love feedback about this.

Reply:
Just a styling correction, on page 53 of the style guide it has an example of a 'selected' row of a table. That is not happening what a row is "selected". https://spark.healthcatalyst.com/docs/DOC-3141

Reply:
Good catch on style. My thought process was you can have a row be selectable and if no checkboxes were present it would look like the selected style on the styles tab now (https://cashmere.azurewebsites.net/styles/table), but if checkboxes were present it would just update the checkbox. But perhaps we do the lighter blue highlight when checkboxes are present and use the darker blue one when no checkbox is present?